python/ql/test/query-tests/Security/CWE-295/paramiko_host_key.py

```diff
@@ -0,0 +1,7 @@
+from paramiko.client import AutoAddPolicy, WarningPolicy, RejectPolicy, SSHClient
+
+client = SSHClient()
+
+client.set_missing_host_key_policy(AutoAddPolicy) # bad
```

Comment:
Can we add test for instances as well `client.set_missing_host_key_policy(AutoAddPolicy())`